### ISSUE TITLE
LFS in GIT submodule is skipping .gitconfig

### ISSUE THREAD

**Issue by panjandrum** [github/git-lfs#922]
Hi guys!
I'm whole day struggle with one big problem. Maybe you will be able to help me?

Lets me explain my situation.
First of all I'm using *lfs-test-server* on localhost with basic authentication.
I have 2 repositories, where second one is used as a GIT submodule in first. In both I'm using the same LFS server.

For simplicity lets assume that it looks like that:
```
first
|-- .git
|-- .gitconfig
`-- second
    |-- .git
    `-- .gitconfig
```

Both `.gitconfig` file looks like:
```
[lfs]
    url = "http://127.0.0.1:9999/"
[lfs "http://127.0.0.1:9999/"]
    access = basic
```

When I'm cloning first one, everything is ok. Files are downloaded from LFS server. However afer i run `git submodule update --init --remote` I have following error:
```
Error accessing media: bin/makedepend (0759e66ca9ae5ab00efa3641c604bd8e6e827f3594b8e57d5055a99c2bd5e18a)

Post file:///home/emaclec/opt/git/remote/sandbox/mtas_ftfw_artifacts.git/info/lfs/objects/batch: unsupported protocol scheme "file"
```

It looks like `.gitconfig` from submodule is not loaded at all. When I add .gitconfig content to `first/.git/modules/second/config` it solves the problem, but in my case I would like to avoid this.

Do you know how to force load `.giconfig` when repo is used as a submodule? Or maybe this is a bug?

Thanks in advance
panjandrum

**Comment by technoweenie** [comment]
Seems like lfs should detect that it's in a submodule and pick the right file. Thanks for the report.

**Comment by technoweenie** [comment]
This should be fixed in v1.1.1: https://github.com/github/git-lfs/releases/tag/v1.1.1

Thanks again!
